File: 1895-minimum-number-of-operations-to-move-all-balls-to-each-box/1895-minimum-number-of-operations-to-move-all-balls-to-each-box.py
class Solution:
    def minOperations(self, boxes: str) -> List[int]:
        b=list(boxes)
        answer=[0]*len(boxes)
        for i in range(len(boxes)):
            ans=0
            for j in range(len(boxes)):
                if i==j:
                    continue
                else:
                    if boxes[j]=='1':
                        ans+=abs(i-j)
            answer[i]=ans
        return answer

File: 1895-minimum-number-of-operations-to-move-all-balls-to-each-box/1895-minimum-number-of-operations-to-move-all-balls-to-each-box.java
class Solution {
    public int[] minOperations(String boxes) {
        int[] answer=new int[boxes.length()];
        for(int i=0;i<boxes.length();i++){
            int ans=0;
            for(int j=0;j<boxes.length();j++){
                if(i==j){
                    continue;
                }
                else if(boxes.charAt(j)=='1'){
                    ans+=Math.abs(i-j);
                }
            }
            answer[i]=ans;
        }
        return answer;
    }
}